docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -53,13 +79,15 @@ sdlManager.getScreenManager().setMenu(Collections.singletonList(cell));
 !@
 
 ### Adding Submenus
-Adding a submenu is as simple as adding subcells to a @![iOS]`SDLMenuCell`!@ @![android, javaSE, javaEE]`SdlMenuCell`!@. The submenu is automatically displayed when selected by the user. Currently menus only support one layer of subcells.
+Adding a submenu is as simple as adding subcells to a @![iOS]`SDLMenuCell`!@ @![android, javaSE, javaEE]`SdlMenuCell`!@. The submenu is automatically displayed when selected by the user. Currently menus only support one layer of subcells. In Core v6.0+ it is possible to set individual submenus to use different layouts such as tiles or lists.
 
 @![iOS]
 ##### Objective-C
+
+###### Pre Core v6.0
```

Comment:
We can just update the code snippets to use the new `init`s as it will still work with head units with pre RPC v6.0. The the tile layout will just not be set.